tests/e2e/CHANGELOG.md

```diff
@@ -0,0 +1,152 @@
+## [2.0.2](https://github.com/algolia/instantsearch-e2e-tests/compare/v2.0.1...v2.0.2) (2022-02-03)
```

Comment:
Do we still need a changelog? This is now a private package, this doesn't seem useful any longer.

Reply:
It's actually more work to prevent the changelog than to have it (you need to ignore it in gitignore), so I kept it. It's indeed arguable whether it's useful since we have git diff too.

Reply:
It actually will be written on every release, just like the other packages :)

Reply:
Ah, I didn't realize we'd be doing one changelog per project.

Unrelated, but do you know how this will look like in [releases](https://github.com/algolia/instantsearch.js/releases)?

Reply:
I hadn't gone through the whole release flow, it's possible only one of the packages will have a GitHub release, but normally it should be one release per tag, and that tag having its changelog. Every package gets its own tag